src/components/application_manager/CMakeLists.txt

```diff
@@ -28,405 +28,110 @@
 # ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 
-if (${CMAKE_SYSTEM_NAME} MATCHES "Windows")
-  find_package(WinOpenSSL REQUIRED)
-endif()
-
-if (VERBOSE)
-    message ("Generating project ${PROJECT_NAME} in ${CMAKE_CURRENT_BINARY_DIR}")
-endif()
+include(${CMAKE_SOURCE_DIR}/tools/cmake/helpers/platform.cmake)
+include(${CMAKE_SOURCE_DIR}/tools/cmake/helpers/sources.cmake)
 
-set (AM_SOURCE_DIR ${COMPONENTS_DIR}/application_manager)
-set (AM_TEST_DIR ${AM_SOURCE_DIR}/test)
+find_package(SDLOpenSSL REQUIRED)
+find_package(SDLSqlite3 REQUIRED)
 
-include_directories (
-  ${COMPONENTS_DIR}/application_manager/include/
-  ${COMPONENTS_DIR}/utils/include/
-  ${COMPONENTS_DIR}/formatters/include/
-  ${COMPONENTS_DIR}/protocol_handler/include/
-  ${COMPONENTS_DIR}/smart_objects/include/
+include_directories(
+  ${COMPONENTS_DIR}/include
```

Comment:
@anosach-luxoft , as I know, we dont need to include it in every list, because its included in root CMakeLists.txt. Or not?